scripts/cat/status.py

```diff
@@ -529,15 +531,14 @@ def change_group_nearness(self, group: CatGroup):
                     entry["near"] = not entry["near"]
 
     # RETRIEVE INFO
-    def get_standing_with_group(self, group: CatGroup) -> list[CatStanding]:
+    def get_standing_with_group(self, group: CatGroup) -> Optional[list[CatStanding]]:
         """
         Returns the list of standings a cat has for the given group.
         """
-        return [
-            entry["standing"]
-            for entry in self.standing_history
-            if entry["group"] == group
-        ]
+        for entry in self.standing_history:
+            if entry["group"] == group:
+                return entry["standing"]
+        return None
```

Comment:
should probably return `[]` since that's easier to handle downstream whilst still being able to say `if value is None`